opba-protocols/xs2a-protocol/src/main/resources/application.yml

```diff
@@ -27,6 +27,40 @@ spring:
 facade:
   urls:
     embedded-ui-base-url: http://localhost:4200
+    # Datasafe security configuration
+    datasafe:
+      # Which keystore to use and how to encrypt keys in it
+      key-store:
+        type: BCFKS
+        encryption-algo: AES128_KWP
+        pbkdf:
+          pbkdf2:
+            algo: PRF_SHA512
+            saltLength: 16
+            iterCount: 10240
+        mac-algo: HmacSHA3_512
+        password-keys-algo: PBEWithHmacSHA256AndAES_128
+      # Which keys to create in keystore
+      keys:
+        enc-key-number: 1
+        sign-key-number: 1
+        # Secret key creation
+        secret:
+          algo: AES
+          size: 128
```

Comment:
Replace AES to 128 in other places:
AES256_CBC

Reply:
All 3 property files changed to 
psu:
      key-pair:
        cipher-algo: AES128_CBC